pkg/skaffold/constants/constants.go

```diff
@@ -63,6 +63,10 @@ var (
 	Service latest.ResourceType = "service"
 
 	DefaultLocalConcurrency = 1
+
+	// UseBuildkit by default for local build
+	// Fallback to UseDockerCLI or Docker API Client when needed
+	UseBuildkit = true
```

Comment:
I'd suggest not bashing `useBuildkit` with this default so that we can actually verify that buildkit is supported.

OR perform the buildkit-supported check into `setDefaultToBuildkit()`

Reply:
`constants.useBuildkit` was used in test  to initialize LocalBuild : `localBuild:  latest.LocalBuild{UseBuildkit: &constants.UseBuildkit},`

i'm afraid it's not easy to check buildkit support  `setDefaultToBuildkit()`(defaults.go), no ctx there